src/metabase/api/cards.clj

```diff
@@ -0,0 +1,53 @@
+(ns metabase.api.cards
+  "Bulk endpoints for Cards"
+  (:require [compojure.core :refer [POST]]
+            [metabase.api.card :as api.card]
+            [metabase.api.common :as api]
+            [metabase.models.interface :as mi]
+            [metabase.util.malli :as mu]
+            [metabase.util.malli.schema :as ms]
+            [toucan2.core :as t2]))
+
+(mu/defn get-dashboards
+  "Get dashboards that a card appears in. The result looks like
+  ```
+  {:card_id 123
+   :dashboards [{:error :unwritable-dashboard :name \"foo\" :id 456}
+                {:error :unreadable-dashboard}
+                {:name \"foo\" :id 678}]}
```

Comment:
nit: feel free to ignore this, but from the name it feels like this function could just return the list of dashboards, and the caller can construct the full data structure